bobtimus/src/loan.rs

```diff
@@ -0,0 +1,30 @@
+use crate::{LiquidUsdt, Rate};
+use elements::bitcoin::Amount;
+use rust_decimal::Decimal;
+
+#[derive(Debug, Clone, serde::Serialize)]
+pub struct LoanOffer {
+    // TODO: Potentially add an id if we want to track offers - for now we just check if an incoming request is acceptable
+    pub rate: Rate,
+
+    #[serde(with = "::elements::bitcoin::util::amount::serde::as_sat")]
+    pub fee_sats_per_vbyte: Amount,
+
+    #[serde(serialize_with = "LiquidUsdt::serialize_to_nominal")]
+    pub min_principal: LiquidUsdt,
+    #[serde(serialize_with = "LiquidUsdt::serialize_to_nominal")]
+    pub max_principal: LiquidUsdt,
+
+    /// The minimum ltv that is acceptable in percent
+    pub min_ltv: Decimal, // TODO: is u64 appropriate here - how is ltv depicted internally?
```

Comment:
I thought I point this out: 

LTV => **L**oan **T**o **V**alue
Since we want to over-collateralize the loan the LTV ratio will be smaller than 0. 

e.g. 
- current rate: 1 BTC = $100
- principal amount = $100
- collateral amount = 2 BTC
--> LTV = 100 / 2*100 = 0.5

The higher this value is the higher the risk for the lender.

which means LTV value should be a floating point number and a max. 

Note: In the loan contract the liquidation is decided according to the exchange rate (https://github.com/comit-network/baru/blob/a7743b538c91ff3f055c3a1f22a14502be7e275b/src/loan.rs#L68) (cc @luckysori ). 
We will need to derive the value we need to put into liquidation function from the agreed max LTV ratio.